<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="rosterTree">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>SN:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="snLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>558639039</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passLE">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>qoscarpass</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::PasswordEchoOnEdit</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="reqRosterBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Request CL</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="msgPTE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>261</width>
      <height>84</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Hello</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>56</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>SN:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="targetSN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>260454</string>
    </property>
   </widget>
   <widget class="QComboBox" name="msgTypeCB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Plain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Auth Req</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Auth OK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Auth Deny</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>You were added</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Remove yourself</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="logsPTE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>421</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setInfoBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Set info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="typingBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Typing</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="requestCLBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Request CL</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="msgStatus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Online</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Away</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DND</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
